std/algorithm.d

```diff
@@ -1605,9 +1605,9 @@ unittest
 Assigns $(D value) to each element of input range $(D range).
 
 Params:
-        range = an $(XREF2 range, isInputRange, input range) that exposes references to its elements
+        range = An $(XREF2 range, isInputRange, input range) that exposes references to its elements
                 and has assignable elements
-        value = assigned to each element of range
+        value = Assigned to each element of range
```

Comment:
Capitalized because otherwise the current stylesheet makes it look like it belongs to the same sentence as the parameter name, which is awkward to read.